### PR TITLE
feat: Simple example that saves a text value to local storage

### LINKED ISSUE #24: Web Storage API
### Task Description
Implement a simple example to showcase web storage api for persistence.

### Task Requirements

- Create a new directory inside "participants" with your name.
- Develop necessary HTML, CSS, and JavaScript files.
- Integrate file system API.
- Update bootcamp.html with your name under `<navigation>`

### Submission Guidelines

- Create a branch: git checkout -b feature/your-feature-name
- Commit changes: git commit -m "Description of changes"
- Push to branch: git push origin feature/your-feature-name
- Create a Pull Request to merge changes into main.

### Additional Guidelines

- Maintain code quality and organization.
- Implement error handling and thorough testing.
- Document external APIs used.

### Resources
[Web Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API)

### PR DESCRIPTION
**Pull request by pfieffer**
### Description

The PR adds a directory for @pfieffer 's assignments and a simple example to demonstrate web storage api (local storage)

### Changes Made

- A simple form that takes a string value to save in the browser's local storage

### Related Issues

closes #24 

### Checklist

- [x] I have tested these changes locally.
- [x] I have added/updated documentation as needed.
- [x] I have reviewed my code for potential issues.
- [x] I have followed the project's coding guidelines.
- [x] I have updated relevant comments and commit messages.

### Screenshots (if applicable)

image
image

### Reviewer Request

@chauchausoup 
@manjilkoirala 

### By submitting this PR, I confirm that

- [x] I have read and understood the project's contributing guidelines.
- [x] This PR is ready for review and merging.
- [x] I understand that automated tests should pass before merging.

### CODE REVIEW

File: participants/pfieffer/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<body>
+    <h1>Web Storage Example</h1>
+    <h2>I can remember what you type below, even if you close the browser &#128521</h1>
+
+    <p>Enter a value:</p>
+    <input type="text" id="inputValue">
+    <button onclick="saveToLocalStorage()">Save</button>
+
+    <p>Saved Value:</p>
+    <div id="savedValue"></div>
```
Comment (manjilkoirala) [inline participants/pfieffer/index.html:17]: You have added Saved Value text 2 times, one in JS and one in HTML. You can remove it from any one place.

**Review by manjilkoirala — COMMENTED** [review]
Overall, the program is working fine🙂

**Comment by krishnemd** [comment]
With @manjilkoirala 's comment and resolving conflict. LGTM.

File: participants/pfieffer/index.html (hunk 1 above)
Comment (pfieffer) [reply participants/pfieffer/index.html:17]: Hi. Thank you for pointing out, I will get rid of the html paragraph

**Comment by pfieffer** [comment]
Made changes according to @manjilkoirala 's feedback.
Rebased with latest main and resolved conflicts.
Please have a look @chauchausoup @manjilkoirala

**Review by manjilkoirala — APPROVED** [review]
Please change your commit from the main branch to the feature branch then merge.

**Comment by pfieffer** [comment]
> Please change your commit from the main branch to the feature branch then merge.

I think its alright to merge the main branch of a fork.
Let me know your thoughts. @manjilkoirala

**Comment by manjilkoirala** [comment]
> > Please change your commit from the main branch to the feature branch then merge.
> 
> I think its alright to merge the main branch of a fork. Let me know your thoughts. @manjilkoirala

It is recommended to commit with another branch. You can use this name: feature:web-storage-api.
Just create a new branch and do some minor changes then push that branch, that's all.

**Review by ankman007 — APPROVED** [review]
Seems good.

**Pull request merged**